### DISCUSSION TITLE
AR mode ("view in your space" button) not works

### DISCUSSION THREAD

**Discussion by pmorshed** [google/model-viewer#4669] [General]
I tried some gltf models with my mobile phone (iPhone 13) through the [editor](https://modelviewer.dev/editor/) (and QR code).
The 3D models shown well, but the augmented reality button ("View in your space") does nothing! why?

**Comment by elalish** [comment]
Duplicate of #4271 - Apparently this issue is editor-specific and AR is working fine on our regular examples?

**Reply by pmorshed** [reply]
I tried with regular examples and the problem still exists. But this trick works for me:
[..if "view in your space" is tapped before the model has finished loading..] 
https://github.com/google/model-viewer/issues/4271#issuecomment-1666858456
